core/commands/storage/challenge/challenge_helper_test.go

```diff
@@ -64,3 +65,13 @@ func TestGenAndSolveChallenge(t *testing.T) {
 		}
 	}
 }
+
+func TestTimeEvaluate(t *testing.T) {
+	tm := TimeEvaluate{}
+	tm.RecordTime("start")
+	time.Sleep(1 * time.Second)
+	tm.RecordTime("1second")
+	time.Sleep(1 * time.Second)
+	tm.RecordTime("2second")
+	t.Log(tm.Report())
```

Comment:
log does not validate test, should check the report contains mentioned strings such as "start", "1second" and "2second"